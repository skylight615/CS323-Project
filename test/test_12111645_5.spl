int test(){
    int a=0;
    a=0;
    int a=0;
    a=0;
    int a=0;
}