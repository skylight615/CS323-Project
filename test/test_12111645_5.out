Error type 11 at Line 8: invoking non-function variable
Error type 5 at Line 9: unmatching type on both sides of assignment
Error type 1 at Line 12: s is used without a definition
Error type 12 at Line 12: indexing by non-integer
Error type 1 at Line 12: g is used without a definition
Error type 5 at Line 12: unmatching types on both sides of assignment
